build/windows/installer/scripts/td-agent-bit-conf-customizer.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/local/bin/ruby
```

Comment:
same comment. File is duplicated from plugin folder. Since they are in the same repo now, should be removed and copied.

Reply:
> same comment. File is duplicated from plugin folder. Since they are in the same repo now, should be removed and copied.

> refactored under common->scripts since these are common both windows and linux agent.